### DISCUSSION TITLE
Windows 10 - Jamulus, Alesis Multimix 4 & ASIO4ALL (v2.15) Sound Issues

### DISCUSSION THREAD

**Discussion by gibbsp1** [jamulussoftware/jamulus#2882] [General]
I am trying to set-up a private server to Jam with a mate. When I connect to either a server created by my mate or a private server created by me, I can hear him but he can't hear me. If my mate connects to a public server he can't hear them either. However, I can...

I'm guess that the problem must be with my mates set-up. He is using the ASIO4ALL (v2.15) driver and his audio interface is an Alesis Multimix 4. 

Has anyone successfully setup Jamulus on Windows 10 with this combination of device and driver?

**Comment by pljones** [comment]
Looking on [Google for that device and ASIO](https://www.google.com/search?q=Alesis+Multimix+4+asio&oq=Alesis+Multimix+4+asio), there appears to be a lot of problems.  It's probably worth taking a look into how others have got it working -- if at all -- in those results, too.

**Comment by dcorson-ticino-com** [comment]
As I read the Alesis website they don't supply an ASIO driver for this device.
ASIO4ALL may be your only possibility, but I have never gotten ASIO4ALL to work on my Win10 setup (not Alesis).
You may end up having to trade for an interface with a native ASIO driver, unfortunately.
